molgenis/molgenis-emx2
# feat(emx2): on column/table change update ref/refbacks automagically

Fixes #4091

Refback is deleted when
- refTable is dropped
- refBack column is dropped
- refBack column is changed to non-reference type

Refback is fixed when
- refTable is renamed
- refBack column is renamed

Ref/refarray is fixed when
- refTable is renamed

how to test:
- see unit tests (for the refTable+ref rename there was already a test that broke)
- use schema editor and play around with this
